sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/sample_copy_model_to.py

```diff
@@ -56,11 +56,11 @@ def sample_copy_model_to(custom_model_id):
     )
     copied_over_model = poller.result()
 
-    print("Model ID: {}".format(model.model_id))
```

Comment:
Mmmm I'm thinking I could update this to use fstrings now too......or maybe I'll open an issue and do all the samples together.